source/scripts/models/Hero.js

```diff
@@ -52,6 +54,10 @@ export default class Hero extends Pixi.Sprite {
         this.timeSinceAttack = this.attackCooldownTime
 
         this.monsterRank = "warrior"
+
+        this.collider = new CircleCollider(this.position, this.scale, this.rotation)
```

Comment:
I really like this idea of refactoring colliders into their own class, which every entity owns.